cms-2016-collision-datasets/code/create_hlt_config_file_records.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Create HLT CMS Configuration Files records.
+"""
+
+import json
+import os
+import re
+import zlib
+
+
+NOTE = (
+    "This file describes the exact setup for the CMS software executable which "
+    "was used in a data-processing step. It is provided only <i>for information "
+    "purposes</i>. Although all the components required to <i>analyse</i> the public "
+    "primary datasets - such as corresponding input data, condition data, software "
+    "version - are provided on this portal, it is not necessarily possible to "
+    "<i>reproduce</i> all the described data-processing steps."
+)
+
+
+RECID_START = 27000
```

Comment:
As said, it would be safer to start at 29000 since we have been having JADE and LHCb records coming for the release in the meantime, and since the CMS data will be most probably published in Q1 2024 only. Is this OK?